### ISSUE TITLE
ensure reproducability with random_state

### ISSUE THREAD

**Issue by joemcglinchy** [radosuav/pyDMS#6]
hello there, thank you for putting together this package! it is most useful. 

I was wondering, would setting a random state for the code be beneficial so as to ensure reproducible results should one need to run data over multiple iterations, say over the quality mask layer to include different values for good pixels? would it be as simple as specifying `np.random.seed(some_integer)` at the beginning of the pyDMS.py file?

**Comment by radosuav** [comment]
Good idea but I don't think that pyDMS.py needs to be modified. The randomness is due to using [BaggingRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.BaggingRegressor.html) and it is possible to [pass parameters to it](https://github.com/radosuav/pyDMS/blob/master/pyDMS/pyDMS.py#L662). So it should be enough to add something like:
`"baggingRegressorOpt":        {"random_state": 1}` 
to `commonOpts` in run_pyDMS.py.

Let me know if that works.
